src/tools/conseillers/clotureConseiller.ts

```diff
@@ -90,6 +91,13 @@ const updateUser = (app) => async (idUser, email) =>
     },
   );
 
+const createConseillersTermines = (app) => async (conseiller, dateFincontrat) =>
+  app.service(service.conseillersTermines).Model.create({
```

Comment:
Je rencontre cette erreur lors du lancement du script :
`error: Cannot read properties of undefined (reading 'Model')
`
Je ne sais pas où j'ai oublié quelque chose (j'ai testé insertOne, updateOne mais ça ne passe pas plus)